src/main/drivers/dshot_command.c

```diff
@@ -182,6 +182,7 @@ void dshotCommandWrite(uint8_t index, uint8_t motorCount, uint8_t command, dshot
 
     uint8_t repeats = 1;
     timeUs_t delayAfterCommandUs = DSHOT_COMMAND_DELAY_US;
+    motorVTable_t vTable = motorGetVTable();
```

Comment:
Should this be updated to:

```suggestion
    motorVTable_t *vTable = motorGetVTable();
```

Along with the other pointer changes?

Reply:
This PR needs to pull in the latest changes from the original PR